docs/jobs/openings/web3.md

```diff
@@ -0,0 +1,34 @@
+## Web3 Developer Opening at Aragon
+
+Aragon is an open source, decentralized project fighting for the freedom to organize. We believe that decentralized organizations can solve the world’s worst problems.
+
+We’re looking for a Web3 Developer to help us consolidate smart contracts with a frontend, we call this position The Glue. Our Web 3 Developer must not only be a Javascript rockstar but also a decentralization evangelist, someone who enjoys researching and testing new Web3 technologies.
+
+### Responsibilities
+
+- Take ownership of the development and maintenance of [aragon.js](https://github.com/aragon/aragon.js), the JavaScript library between smart contracts and the native web experience.
```

Comment:
the JavaScript ~~library~~ interface

Reply:
Should we then also change the repo description, since that says `A JavaScript library for interacting with Aragon`?

Reply:
I'd keep interface, the difference is negligible anyway and it is, in fact, a library